drivers/store/memory/cache.go

```diff
@@ -50,34 +51,61 @@ func startCleaner(cache *Cache, interval time.Duration) {
 	go cleaner.Run(cache)
 }
 
-// Counter is a simple counter with an optional expiration.
+// Counter is a simple counter with an expiration.
 type Counter struct {
```

Comment:
no sure why we should export this type as public since the memory store and counter are in the same package, same for these methods, I would keep them private IMHO.

Reply:
I have mixed feelings about that. On one hand you're right, it doesn't provide any additional features and it should be private. On the other hand, I hate when developers of a library make everything private so when you want to reuse a components, you either fork the library of copy / paste the code.

So no strong opinion here, but if we make these private (it was public before this pull requests), I would like to do that on the `v4`.

What do you think ?

Reply:
👍 let's keep it public for that moment.